buildSrc/src/gen-execution.kt

```diff
@@ -31,70 +230,8 @@ fun LangData.execution(nickname: String, configName: String, outDir: File) {
 	import $basePackage.validateExe
 	import org.jdom.Element
 
-	class ${languageName}RunConfigurationFactory(type: ${languageName}RunConfigurationType) : ConfigurationFactory(type) {
-		override fun createTemplateConfiguration(project: Project) = ${languageName}RunConfiguration(project, this)
-	}
-
-	object ${languageName}RunConfigurationType : ConfigurationType {
-		internal val factory = ${languageName}RunConfigurationFactory(this)
-		private val factories = arrayOf(factory)
-		override fun getIcon() = TTIcons.$constantPrefix
-		override fun getConfigurationTypeDescription() = TTBundle.message("$nickname.run-config.description")
-		override fun getId() = "${constantPrefix}_RUN_CONFIG_ID"
-		override fun getDisplayName() = TTBundle.message("$nickname.name")
-		override fun getConfigurationFactories() = factories
-	}
-
-	class ${languageName}RunConfiguration(
-		project: Project,
```

Comment:
The constructor parameter will have a corresponding field only when prefixed with `val` or `var`: https://kotlinlang.org/docs/reference/classes.html#constructors

Reply:
This is the old kotlin code which is deleted.

Reply:
I thought you made the two unnecessary fields because you didn't understand the constructor syntax, therefore I'm leaving this comment

Reply:
oh that's why. thank you